flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGeneratorTest.java

```diff
@@ -785,10 +785,19 @@ public void testSlotSharingOnAllVerticesInSameSlotSharingGroupByDefaultEnabled()
 		final List<JobVertex> verticesSorted = jobGraph.getVerticesSortedTopologicallyFromSources();
 		assertEquals(4, verticesSorted.size());
 
-		final JobVertex source1Vertex = verticesSorted.get(0);
-		final JobVertex source2Vertex = verticesSorted.get(1);
-		final JobVertex map1Vertex = verticesSorted.get(2);
-		final JobVertex map2Vertex = verticesSorted.get(3);
+		JobVertex source1Vertex = null, source2Vertex = null, map1Vertex = null, map2Vertex= null;
+		for (int i = 0; i < 4; i++)
+		{
+			JobVertex vertex = verticesSorted.get(i);
+			if (vertex.getName().equals("Source: source1"))
```

Comment:
if body should always be surrounded by braces.

Reply:
I'd prefer to change the matching to `if (vertex.getName().contains("source1"))`.